tests/acceptance/SendingStatusCest.php

```diff
@@ -0,0 +1,66 @@
+<?php
+
+namespace MailPoet\Test\Acceptance;
+
+require_once __DIR__ . '/../DataFactories/Settings.php';
+require_once __DIR__ . '/../DataFactories/Newsletter.php';
+require_once __DIR__ . '/../DataFactories/Subscriber.php';
+
+use Codeception\Util\Locator;
+use MailPoet\Test\DataFactories\Settings;
+use MailPoet\Test\DataFactories\Newsletter;
+use MailPoet\Test\DataFactories\Subscriber;
+
+class SendingStatusCest {
+  function newsletterSendingStatus(\AcceptanceTester $I) {
+    $I->wantTo('Check the sending status page for a standard newsletter');
+    // Having a standard newsletter sent to 2 subscribers
+    $lucky_subscriber = (new Subscriber)
+      ->withFirstName('Lucky')
+      ->withLastName('Luke')
+      ->create();
+    $unlucky_subscriber = (new Subscriber)
+      ->withFirstName('Unlucky')
+      ->withLastName('John')
+      ->create();
+    $newsletter = (new Newsletter)
+      ->withSubject('Testing newsletter sending status')
+      ->withSentStatus()
+      ->withSendingQueue([
+        'count_processed' => 2,
+        'count_total' => 2,
+      ])
+      ->beingSentToSubscriber($lucky_subscriber)
+      ->beingSentToSubscriber($unlucky_subscriber, [
+        'failed' => 1,
+        'error' => 'Oh no!',
+      ])
+      ->create();
+    // When I visit the newsletters page
+    $I->login();
+    $I->amOnMailPoetPage('Emails');
+    $I->waitForText($newsletter->subject);
+    // I click on the "Sent to 3 of 3" link
+    $I->click('[data-automation-id="sending_status_' . $newsletter->id . '"]');
+    // Check I am on the sending status page
+    $I->seeInCurrentUrl('?page=mailpoet-newsletters#/sending-status/' . $newsletter->id);
```

Comment:
Why do we need this check?

Reply:
Just to make sure the link is working.

Reply:
Wouldn't that be obvious from the next check(s)?

Reply:
Yeah, you're right. I will remove that line.